### ISSUE TITLE
Data loading with `fileURL` blocked by CORS on Firefox

### ISSUE THREAD

**Issue by thewtex** [Kitware/vtk-js#1000]
Visiting:

https://kitware.github.io/vtk-js/examples/GeometryViewer/GeometryViewer.html?fileURL=https://data.kitware.com/api/v1/item/59de9de58d777f31ac641dc5/download

On Firefox 65.0 (Linux), the data does not load with:

```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://data.kitware.com/api/v1/item/59de9de58d777f31ac641dc5/download. (Reason: CORS request did not succeed).
```

in the debug console.

**Comment by thewtex** [comment]
Issue number 1000! :sparkles:

**Comment by floryst** [comment]
I cannot reproduce this on Firefox 65.0 on Linux, as the dataset loads for me. Can you try again?

**Comment by thewtex** [comment]
@floryst I tried again, and the issue persists :-(. Firefox 65.0 on Ubuntu 18.04.

**Comment by floryst** [comment]
Does this occur on chrome for you as well? Any possible extensions interfering with traffic?

**Comment by thewtex** [comment]
Chrome works well.

**Comment by floryst** [comment]
Update: this was caused by Privacy Badger somehow learning that "data.kitware.com" is a tracking domain, despite it not being one.

**Comment by thewtex** [comment]
Perhaps we need to serve `https://data.kitware.com/.well-known/dnt-policy.txt` with the content of [dnt-policy-1.0.txt](https://raw.githubusercontent.com/EFForg/dnt-policy/master/dnt-policy-1.0.txt) since this is checked:

> - [ ] Check if third-party domains comply with EFF's [Do Not Track policy](https://www.eff.org/dnt-policy)

(this setting is enabled by default).

@manthey @mgraurer @zachmullen is this possible?

**Comment by zachmullen** [comment]
If sysadmin wants to set that up in the reverse proxy, it's fine by me. It's out of scope for Girder itself.

**Comment by thewtex** [comment]
I will contact sys-admin...
